server/src/core/server/services/users/helpers.ts

```diff
@@ -104,3 +104,65 @@ export function checkForNewUserEmailDomainModeration(
   }
   return null;
 }
+
+interface EmailIsAliasResult {
+  base?: {
+    value: string;
+    start: string;
+    domain: string;
+  } | null;
+  alias?: {
+    value: string;
+    start: string;
+    domain: string;
+  } | null;
+  isAlias: boolean;
+}
+
+export function emailIsAlias(email?: string | null): EmailIsAliasResult {
+  if (!email) {
+    return {
+      base: null,
+      alias: null,
+      isAlias: false,
+    };
+  }
+
+  const split = email.split("@");
```

Comment:
just wondering if the naming could be clearer here too? Maybe clarifying that this is `emailSplit` would help since there's also the `aliasSplit` below.

Reply:
fixed